packages/analytics/shippers/elastic_v3/server/src/server_shipper.test.ts

```diff
@@ -154,7 +154,7 @@ describe('ElasticV3ServerShipper', () => {
     fakeSchedulers((advance) => {
       shipper.reportEvents(events);
       shipper.optIn(false);
-      setLastBatchSent(Date.now() - 10 * MINUTES);
+      setLastBatchSent(Date.now() - 10 * SECONDS);
```

Comment:
Similar to my previous comment; the comment string on line 153 also needs to be revised to talk of 10 seconds, not minutes, right?